### ISSUE TITLE
[FEAT] Create a comparison for lat-long great circle distance

### ISSUE THREAD

**Issue by ADBond** [moj-analytical-services/splink#1039]
### Is your proposal related to a problem?
Thanks to #1038 I've realised that we do not have a ready-made comparison in the `comparison_library` for evaluating the `distance_in_km_level` at various thresholds.

### Describe the solution you'd like
An addition to the `comparison_library` that outputs several levels at different km thresholds. E.g.
```py
cl.distance_in_km_at_thresholds(lat_col="lat", long_col="long", km_thresholds=[1, 5, 10])
```
would give levels:
* distance <= 1 km
* distance <= 5 km
* distance <= 10 km
* else

### Describe alternatives you've considered
Can do things manually now, but this feels like a pretty common use-case

**Comment by aflaxman** [comment]
Thanks for all the amazing work you and the splink team have done!  This sounds like a great feature, but it reminded me of one terminology thing that has been bugging me about some of your comparison level names --- would it be more accurate to call the level "1 km < distance <= 5 km"?  (i've been meaning to ask that about Levenstein comparison level names, too.)  Thanks again!

**Comment by NickCrews** [comment]
> would it be more accurate to call the level "1 km < distance <= 5 km"

Yes, that's exactly what is happening. You're right, the name is a bit incorrect. Same for levenshtein, etc

**Comment by ADBond** [comment]
This is a good point @aflaxman - in each case the levels are implicitly excluding any previous levels (so as you say, the level is really '1km < distance <= 5km', or perhaps closer to what is actually evaluated '(distance <= 5km) AND NOT (distance <= 1km)'.
I do think it would be useful to have more holistically accurate level names, but there are some challenges to this. It would be fairly easy to adjust things in the case of the comparison library to get more descriptive names, but if we are building up a comparison [by combining in-built comparison levels](https://moj-analytical-services.github.io/splink/topic_guides/customising_comparisons.html#method-2-comparisonlevels), then each level does not directly know about the others, as they are created individually. We would need to think about where this would propagate through, although in such a case there would be a lot of 'and not' statements which may make things _less_ readable.

**Comment by RossKen** [comment]
This has been solved by #1117, merged into master and will be part of the next release.
